modules/caddyhttp/fileserver/caddyfile.go

```diff
@@ -168,6 +170,17 @@ func (fsrv *FileServer) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 			falseBool := false
 			fsrv.CanonicalURIs = &falseBool
 
+		case "file_limit":
```

Comment:
This needs to be moved to line 133-ish to be inside of `browse`

Reply:
Oh yes,  I misread your config and put it under the first browse. Updating now.